taxonomy-manager-common/src/main/java/com/digirati/taxman/common/taxonomy/CollectionModel.java

```diff
@@ -0,0 +1,26 @@
+package com.digirati.taxman.common.taxonomy;
+
+import com.digirati.taxman.common.rdf.RdfModel;
+import com.digirati.taxman.common.rdf.annotation.RdfConstructor;
+import com.digirati.taxman.common.rdf.annotation.RdfContext;
+import com.digirati.taxman.common.rdf.annotation.RdfType;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.vocabulary.DCTerms;
+import org.apache.jena.vocabulary.SKOS;
+
+@RdfType("http://www.w3.org/2004/02/skos/core#Collection")
+@RdfContext({"skos=" + SKOS.uri, "dcterms=" + DCTerms.NS})
+public class CollectionModel implements RdfModel {
```

Comment:
I've added this class as part of the autocomplete PR, so we should probably make sure we're using the same one in both changes